man/systemd-boot.xml

```diff
@@ -542,9 +542,14 @@
     <variablelist class='efi-variables'>
       <varlistentry>
         <term><varname>SYSTEMD_BOOT_LOG_LEVEL</varname></term>
-        <listitem><para>Logging level to use. One of <literal>fatal</literal>, <literal>error</literal>,
-        <literal>warning</literal> (default), <literal>info</literal>, <literal>debug</literal> or
-        <literal>trace</literal>.</para></listitem>
+        <listitem>
+          <para>Logging level to use. One of <literal>fatal</literal>, <literal>error</literal>,
+          <literal>warning</literal> (default), <literal>info</literal>, <literal>debug</literal> or
+          <literal>trace</literal>.</para>
+
+          <para>Note that all log messages will be sent to a QEMU debugcon on port 0x402 if run
+          under a VM regardless of the current log level.</para>
```

Comment:
Hmm, the description in the commit message is much more accessible. I would have no idea how to use this based on the man page description. Please add something like this:
"Hint: use `-chardev stdio,id=debugcon -device isa-debugcon,iobase=0x402,chardev=debugcon` as a parameter to qemu."